client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail/workflow-detail.component.html

```diff
@@ -106,7 +106,7 @@ <h2 *ngIf="workflow">
                             <div class="inner-table">
                                 <div *ngIf="!state.next_states.length">-</div>
                                 <div *ngIf="state.next_states.length">
-                                    <os-comma-separated-listing [list]="state.next_states">
+                                    <os-comma-separated-listing [list]="sortedNextStates(state)">
```

Comment:
This can cause performance issues because this gets calculated on every change detection cycle and the underlying method performs a sort. 

However because the next states might be quite small in most cases I approve.